internal/stackdriver/stackdriver.go

```diff
@@ -44,6 +44,46 @@ func NewAPIClient(ctx context.Context, project string) (*API, error) {
 	}, nil
 }
 
+// RequestCount count returns the number of requests for the given offset and
+// query.
+func (a *API) RequestCount(ctx context.Context, query Query, offset time.Duration) (int64, error) {
+	query = addFilterToQuery(query, "metric.type", requestCount)
+	endTime := time.Now()
+	startTime := endTime.Add(-1 * offset)
+
+	it := a.MetricClient.ListTimeSeries(ctx, &monitoringpb.ListTimeSeriesRequest{
+		Name:   "projects/" + a.Project,
+		Filter: query.Query(),
+		Interval: &monitoringpb.TimeInterval{
+			StartTime: timestamp.New(startTime),
+			EndTime:   timestamp.New(endTime),
+		},
+		Aggregation: &monitoringpb.Aggregation{
+			AlignmentPeriod:    duration.New(offset),
+			PerSeriesAligner:   monitoringpb.Aggregation_ALIGN_DELTA,
+			GroupByFields:      []string{"resource.labels.service_name"},
+			CrossSeriesReducer: monitoringpb.Aggregation_REDUCE_SUM,
+		},
+	})
```

Comment:
need to handle ExecutionErrors field
https://godoc.org/google.golang.org/api/monitoring/v3#ListTimeSeriesResponse

it seems sometimes rpc succeeds, but api returns errors in a field.

Reply:
With the new library, this is not the case (see my previous comment)